.env.test

```diff
@@ -1,2 +1,2 @@
 REACT_APP_WEBMAP_DOMAIN=http://test.treetracker.org
-
+REACT_APP_QUERY_API_ROOT=https://dev-k8s.treetracker.org/query
```

Comment:
Temporarily use dev env for the time being, I will deploy the query API to test env in the near future.